test/integration/cls.test.js

```diff
@@ -33,6 +33,18 @@ if (current.dialect.supports.transactions) {
     });
 
     describe('context', () => {
+      it('does not use continuation storage on manually managed transactions', function() {
+        const self = this;
+
+        return Sequelize._clsRun(() => {
```

Comment:
Fix indentation, its breaking build

Reply:
I'm confused a bit here, the indentation looks fine.  Can you please point me to where the rules are being violated?

Reply:
just run `npm run lint` you will know